rpm/firejail.spec

```diff
@@ -0,0 +1,38 @@
+Name: firejail
+Version: 0.9.62
+Release: 1
+Summary: Linux namepaces sandbox program
+License: GPLv2+
+Group: Development/Tools
```

Comment:
Should not be used: https://sailfishos.org/wiki/Software_Packaging#RPM_packaging_Guidelines

Reply:
`Group` removed